apps/ledger-live-desktop/jest.config.js

```diff
@@ -37,6 +37,13 @@ const defaultConfig = {
 };
 
 module.exports = {
+  collectCoverage: true,
+  collectCoverageFrom: [
+    "src/**/*.{ts,tsx}",
+    "!src/**/*.test.{ts,tsx}",
+    "!src/**/*.spec.{ts,tsx}",
```

Comment:
@KVNLS There is 1 .spec.ts file in the desktop app, none in the mobile one. This will need to be changed as it is inconsistent, I'll write guidelines we can agree on and reach out to the team in question later.